crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -291,7 +303,8 @@ def noderepocheck
   rescue Crowbar::Error::StartStepRunningError,
          Crowbar::Error::StartStepOrderError,
          Crowbar::Error::EndStepRunningError,
-         Crowbar::Error::SaveUpgradeStatusError => e
+         Crowbar::Error::SaveUpgradeStatusError,
+         StandardError => e
```

Comment:
No, I think you need to handle `StandardError` separately and re-raise it. Otherwise we do not have full trace in the production.log